images/libvirtd/libvirtd.sh

```diff
@@ -6,16 +6,9 @@ fatal() { echo "FATAL: $@" >&2 ; exit 2 ; }
 [[ -f /host/var/run/libvirtd.pid ]] && fatal "libvirtd seems to be running on the host"
 brctl show | grep virbr >/dev/null && fatal "libvirtd bridges are present on the host"
```

Comment:
But wouldn't this mean that after a container restart the container does not come up again?